src/test/java/com/exasol/hamcrest/SqlFragmentRenderResultMatcher.java

```diff
@@ -54,6 +57,14 @@ public boolean matchesSafely(final Fragment fragment) {
             final DropTableRenderer renderer = new DropTableRenderer(this.config);
             ((DropTableFragment) root).accept(renderer);
             this.renderedText = renderer.render();
+        } else if (root instanceof CreateSchemaFragment) {
```

Comment:
We are approaching the point where this part should be done via reflection. This method contains a whole lot of tight coupling.

Reply:
done